content/docs/reference/service-urls.md

```diff
@@ -0,0 +1,287 @@
+---
+title: Service URLs
+id: service-urls
+description: This page covers all of Pomerium's service URL settings, including authenticate, authorize, and databroker external and internal service URLs.
+keywords:
+  - authenticate service url
+  - authenticate internal service url
+  - authenticate_service_url
+  - authenticate_internal_service_url
+  - authorize service url
+  - authorize_service_url
+  - authorize internal service url
+  - authorize_internal_service_url
```

Comment:
Let's include databroker keywords here too, for consistency.

Reply:
I'm also not sure if adding the config file key and environment variable variants as keywords here is helpful for internal search, but that's the reasoning behind adding these.